internal/db/model.go

```diff
@@ -67,6 +69,18 @@ func (a Attribute) ParseFromJson(value interface{}) (interface{}, error) {
 			return nil, fmt.Errorf("%w: expected string/text got %T", ErrValue, value)
 		}
 		return s, nil
+	case EmailAddress:
+		emailAddressString, ok := value.(string)
+	    if ok {
+	    	//https://www.alexedwards.net/blog/validation-snippets-for-go#email-validation
```

Comment:
factor this out into a separate package and import it here (maybe `awans.org/aft/datatypes`..)

Reply:
eventually that's where we can put a bunch of core "application" types like email when we have a proper datatype interface, but I still prefer to at least not just inline this here..